LongWarOfTheChosen/Src/LW_Overhaul/Classes/X2TargetingMethod_LWRocketLauncher.uc

```diff
@@ -114,8 +114,11 @@ function Update(float DeltaTime)
 		DrawAOETiles(Tiles);
 
 		//update expected scatter amount display
-		vMouseCursorPos = class'UIUtilities_LW'.static.GetMouseCoords();
-		ScatterAmountText.SetPosition(vMouseCursorPos.X + 2, vMouseCursorPos.Y + 14); // this follows cursor
+		// Start Issue #150 : Make the "tooltip" not rely on the  existance of the mouse pointer
+		//vMouseCursorPos = class'UIUtilities_LW'.static.GetMouseCoords();
```

Comment:
No need to follow CHL conventions here. I would just remove this line. I hate the accumulation of commented out code. Since it's all in git, we have a history to track what's happened.

Reply:
Also, don't need the "Start Issue" comment, although if you want to keep it there, that's fine too.

Reply:
Sure, old habits die hard, remember the 😠 Robo for my lack of comments etc back when I first started contributing to the CHL 😁 . Also also, somewhat a direct lift from I did for Iridar, where the commenting out was as much to help him know where to copypasta the code snippet!
Actual point: Shiremct is taking this on board for his mod which uses LW2 scatter, and picked up on something I missed, BlasterLauncher scatter. Missed that because BL targeting isn't so fatal to controllers (so `SetCursorLocation()`), just the scatter estimate "tooltip" issue. I'll add that to the branch/PR this weekend sometime...